typespec-tests/tsp/protocol-api-convenient-api.tsp

```diff
@@ -97,6 +97,19 @@ model ResourceJ {
   type: string;
 }
 
+@trait
+model ListTrait {
+  @traitContext(TraitContext.List)
+  reqeust: {
+    @traitLocation(TraitLocation.Parameters)
+    properties: {
+      @projectedName("client", "maxPageSize")
```

Comment:
Is `maxPageSize` SDK's conventional name for the pageSize? If service defines otherwise, we need to tell them to use `@projectedName` right?

Reply:
Yes. Ideally it should have serialziedName = maxpagesize. For those not define as such, we allow them to use projectedName to maxPageSize (as we cannot change their serialziedName)